examples/devices/sensor_tag_ble_dell5k_mqtt_iotcc.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------#
```

Comment:
As you are making the new folder I`ll prefer if you can move the entities in the specific folder.